rotkehlchen/tests/unit/decoders/test_cowswap.py

```diff
@@ -0,0 +1,366 @@
+import pytest
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryBaseEntry
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.chain.ethereum.modules.cowswap.constants import CPT_COWSWAP
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.constants.assets import A_CRV, A_ETH, A_USDC, A_USDT, A_WBTC, A_WETH
+from rotkehlchen.fval import FVal
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [[
+    '0x0D2f07876685bEcd81DDa1C897f2D6Cacc733fc1',
```

Comment:
Why add 2 addresses? Most of the times the way this works is many user addresses are coupled into one coswap solver transaction like this. Most likely the different swappers are different parties/people.

Reply:
I know that in 99% cases swaps will be for different people. However I included 2 addresses in each test to make sure that different combinations of decoded swaps don't break anything.

In my opinion when testing for 2 addresses the quality of tests doesn't reduce, but we also check that in case there will be multiple addresses tracked nothing will break. Or am I wrong?

Reply:
It's hard to read the test and understand the testing scenario

Reply:
Changed that